dts/bindings/psi5/psi5-controller.yaml

```diff
@@ -0,0 +1,153 @@
+      description: |
+        Determines the channel operation mode. When set to true, the channel operates in
+        asynchronous mode with only the receive function active. When set to false,
+        the channel operates in synchronous mode with both transmit and receive functions active.
+
+    period-sync-pulse-us:
+      type: int
+      required: true
+      description: |
+        Specifies the period of the internally generated synchronization pulses, measured in
+        microseconds (us). This value determines the length of each synchronization pulse used
+        in the system.
+
+    pulse-width-0-us:
+      required: true
+      type: int
+      description: |
+        Specifies the duration of the pulse width for a data bit value '0', measured in
+        microseconds (us).
+
+    pulse-width-1-us:
+      required: true
+      type: int
+      description: |
+        Specifies the duration of the pulse width for a data bit value '1', measured in
+        microseconds (us).
+
+    decoder-start-offset-us:
+      type: int
+      required: true
+      description: |
+        Specifies the duration for which the Manchester decoder remains inactive after the
+        falling edge of a synchronization pulse, measured in microseconds (us). This value
+        determines the delay before the decoder starts processing incoming signals again.
+
+    tx-mode:
+      type: string
+      required: true
+      enum:
+        - short-frame-31
+        - short-frame-5
+        - long-frame-31
+        - long-frame-5
+        - x-long-frame-31
+        - x-long-frame-5
+        - xx-long-frame
+        - non-standard-frame
+      description: |
+        Specifies the transmitter mode. Each mode defines the frame length and
+        the start condition for data transmission:
+          - short-frame-31: Short Frame (V1.3) with 31 "1s" as the start condition
+          - short-frame-5: Short Frame (V1.3) with 5 "0s" as the start condition
+          - long-frame-31: Long Frame (V1.3) with 31 "1s" as the start condition
+          - long-frame-5: Long Frame (V1.3) with 5 "0s" as the start condition
+          - x-long-fram-31: X-Long Frame (V1.3) with 31 "1s" as the start condition
+          - x-long-frame-5: X-Long Frame (V1.3) with 5 "0s" as the start condition
+          - xx-long-frame: XX-Long (V2.0)
+          - non-standard-frame: Non Standard Length
+
+    num-rx-buf:
+      type: int
+      required: true
+      description: |
+        Specifies the maximum number of receive buffers used for storing PSI5 messages.
+        The value can range from 1 to 32, determining how many messages can be stored in
+        the buffer at any given time.
+
+    rx-bitrate-kbps:
+      type: int
+      required: true
+      enum:
+        - 125
+        - 189
+      description: |
+        Selects the receive message bitrate in kbps. This setting determines the speed at
+        which data is received.
+
+    array-slot-duration-us:
```

Comment:
I think it's better to use child binding for the slot, should we consider to use it instead of the array?

Reply:
I used a child binding for these config for channel rx slot

Reply:
I updated to use child binding for these config for channel rx slot